cmd/security-proxy-setup/res/configuration.toml

```diff
@@ -50,53 +50,63 @@ TokenPath = "res/resp-init.json"
 CACertPath = "res/EdgeXFoundryCA/EdgeXFoundryCA.pem"
 SNIS = ["edgex-kong"]
 
-[Clients]
-  [Clients.Coredata]
+[Services]
```

Comment:
Why change from Clients to Services?

Reply:
Due to the comment mentioned earlier.  For identification purposes.  ClientInfo does not provide the capability to offer the unique struct instance name.  Rather then modifying the ClientInfo which is uses in numerous places I chose to use what the existing Security services API uses which is a custom type.   This type "struct" is nearly identical in shape to the KongType.

Reply:
I don't consider this resolved until I understand your answer to my question.  What "comment mentioned earlier" are you referring to?

Reply:
@michaelestrin sorry I was referring to the previous comment where i mentioned the use of the ServiceInfo abstraction struct previously in the security api's.  https://github.com/edgexfoundry/security-api-gateway/blob/master/internal/pkg/edgexproxy/service.go .  In the security api there is the use of the KongService struct.  I took the similar approach and used the same pattern used in that API.   The underlying symptom here is there is no logical name for any of the given afore mentioned clientInfo toml configuration entries.  The pattern used in the Security API is the inclusion of the logical name.  We need the name of the service so we can use that to bind not only client but service call.  Rather then including a seperate list of unique names in yet another configuration struct, the pattern reuses the "name" for both those entries.

Reply:
>there is no logical name for any of the given afore mentioned clientInfo toml configuration entries.

There is -- the service's name is embedded in the encompassing toml chapter name.  If we depart from that convention, we should do so consistently throughout all of the services.

I feel like I'm missing something.